src/main/java/mate/academy/spring/model/User.java

```diff
@@ -45,6 +62,8 @@ public void setPassword(String password) {
     public String toString() {
         return "User{"
                 + "id=" + id
-                + ", email='" + email + '\'' + '}';
+                + ", email='" + email + '\''
+                + ", roles=" + roles
+                + '}';
```

Comment:
```suggestion
                + ", email='" + email + '\''
                + '}';
```
this field can lead to an error  LazyInitializationException

Reply:
I'm not agree: by default in method "register" in AuthenticationServiceImpl class we set the Role "USER" to each new user.